tests/integrationv2/providers.py

```diff
@@ -55,7 +55,7 @@ def __init__(self, options: ProviderOptions):
 
     def setup_client(self, options: ProviderOptions):
         self.ready_to_test = 'listening on lo'
-        tcpdump_filter = f"dst port {options.port}"
+        tcpdump_filter = "dst port {}".format(options.port)
```

Comment:
Thanks! Sorry for all those syntax errors with the new string formatting.

Reply:
no worries!